content/liveobjects/storage.textile

```diff
@@ -0,0 +1,62 @@
+---
+title: Object storage
+meta_description: "Learn about LiveObjects object storage."
+product: liveobjects
+languages:
+  - javascript
+---
+
+<aside data-type='experimental'>
+<p>LiveObjects is currently Experimental. Its features are still in development and subject to rapid change.</p>
+</aside>
+
+h2(#default-persistence). Default object storage - 90 days
+
+Ably durably stores all objects on a channel for 90 days by default. If the data is not updated within 90 days, it will automatically expire. After expiry, the channel is reset to its initial state and only includes an empty "root object":/docs/liveobjects/concepts/objects#root-object.
+
+<aside data-type='note'>
+<p>If you're interested in a configurable default object storage duration, please "get in touch":/contact with your requirements.</p>
+</aside>
+
+h2(#outside-persistence). Store objects outside of Ably
+
+You can store your objects outside of Ably by obtaining the channel objects via the "REST API":/docs/liveobjects/rest-api-usage#fetching-objects and storing them in your own database. This is useful if you want to keep a permanent record of the objects on a channel or if you want to perform analytics on the data.
+
+In order to receive notifications when the objects on a channel are updated, use "inband objects":/docs/liveobjects/inband-objects to receive updates as regular channel messages.
+
+<aside data-type='note'>
+<p>If you're interested in exporting objects to your own systems using "integration rules":/docs/integrations/webhooks, please "get in touch":/contact with your requirements.</p>
+</aside>
+
+h2(#operation-persistence). Operation storage
+
+When you update an object, the change is expressed as an "operation":/docs/liveobjects/concepts/operations that is sent as an "object message":/docs/metadata-stats/stats#messages on the channel. Like all messages, Ably stores object messages for 2 minutes by default.
+
+This means that if a client disconnects from Ably for a short period of time, it can automatically retrieve any operations it may have missed when it reconnects. If a client disconnects for longer than 2 minutes, the client will be sent the latest state of the objects on the channel (which are durably stored for 90 days) when it reconnects, ensuring the client remains fully synchronized.
+
+Operations themselves are not included in the "history":https://ably.com/docs/storage-history/history or "rewind":/docs/channels/options/rewind backlog of a channel. Instead, you should interact with objects directly via the client library.
+
+h2(#limits). Object count and size limits
+
+You can store up to 100 distinct objects on a "channel":/docs/pricing/limits#channel .
```

Comment:
Similar comment about the object limit

Reply:
Fixed in: [d57f9d8](https://github.com/ably/docs/pull/2591/commits/d57f9d8eb49eca7a072165f2a4304feb5c440f9b)